core/src/main/java/org/elasticsearch/index/mapper/DocumentParser.java

```diff
@@ -186,6 +186,7 @@ private static void reverseOrder(ParseContext.InternalParseContext context) {
         }
     }
 
+    // TODO Should we remove this feature ? It seems that it's not tested nor used.
```

Comment:
+1 it looks like there are some remainders of the boost field mapper that we removed a while ago (eg. there is also ParseContext.setDocBoost).